dependencies.props

```diff
@@ -30,6 +31,7 @@
     <ExternalExpectedPrerelease>beta-24727-00</ExternalExpectedPrerelease>
     <ProjectNTfsExpectedPrerelease>beta-24926-00</ProjectNTfsExpectedPrerelease>
     <StandardExpectedPrerelease>beta-24919-01</StandardExpectedPrerelease>
+    <SniExpectedPrerelease>beta-24926-02</SniExpectedPrerelease>
```

Comment:
Yes. It's the prefix generated in sni repo.
It will change with preview, release servicing etc.